subprojects/docs/src/samples/userguide/organizeBuildLogic/customPluginWithNestedDsl/build.gradle

```diff
@@ -0,0 +1,39 @@
+class Person {
+    String name
+}
+
+class GreetingPluginExtension {
+    String message
+    final Person greeter
+
+    @javax.inject.Inject
+    GreetingPluginExtension(ObjectFactory objectFactory) {
```

Comment:
I thought that the generation of Closure-taking methods happens automatically for first-level domain objects. Or did this behavior change?